cfbs/commands.py

```diff
@@ -372,11 +372,15 @@ def update_command(non_interactive=False):
             if key == "steps":
                 # same commit => user modifications, don't revert them
                 if commit_differs:
-                    ans = input(
-                        "Module %s has different build steps now\n" % module["name"]
-                        + "old steps: %s\n" % module["steps"]
-                        + "new steps: %s\n" % index_info["steps"]
-                        + "Do you want to use the new build steps? [y/N]"
+                    ans = (
+                        input(
+                            "Module %s has different build steps now\n" % module["name"]
+                            + "old steps: %s\n" % module["steps"]
+                            + "new steps: %s\n" % index_info["steps"]
+                            + "Do you want to use the new build steps? [y/N]"
+                        )
+                        if not non_interactive
+                        else "yes"
```

Comment:
Good old 
```
if non_interactive:
  ans = "yes"
else:
  ans = input(...)
```
would be better here, AFAICT.

Reply:
Yep, agreed: https://github.com/cfengine/cfbs/pull/93/commits/75dba0e8df1c18db8933c4b13f8653ebdd02f588